### ISSUE TITLE
REMOVE deep-diff dependency

### ISSUE THREAD

**Issue by renatobenks** [evgenyrodionov/redux-logger#273]
The deep diff it's horrible and so bad, I take a look on source code about this, and this dependency it's not so good like `redux-logger`, so, I'm working in a own `deep-diff` to `redux-logger` to improve this and remove this dependency, what do you  think @evgenyrodionov?

Please, considering my PR when I'm done, improve that feature, removing this dependency! 🗑

**Comment by evgenyrodionov** [comment]
Maybe we can switch to another dependency? To be honest, we don’t want maintain own code which doesn’t relate to logger in general.

**Comment by antoniopresto** [comment]
deep-diff is causing crashes on redux-logger when passing longs action values, like long route query strings, for example. but redux-logger seems to suppress the error, I'm investigating this.

**Comment by ingro** [comment]
Yeah, I had to set `diff` to false in the options, my app wasn't working anymore in development and redux-logger wasn't emitting any error...

**Comment by grushetsky** [comment]
@renatobenks, FYI, we might consider the overhaul of diff logger in the next major version of Redux Logger (#292).

**Comment by thiamsantos** [comment]
I just made my own version of deep-diff, I called it [ddiff](https://github.com/thiamsantos/ddiff). The test suite is from deep-diff but the entire codebase was written from scratch. Maybe redux-logger could use it. I would love to hear feedback.

**Comment by eranimo** [comment]
Hi—new maintainer here. We're looking at revamping the diff feature, and we probably will move away from this dependency.
